src/components/ec-radio-btn/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './ec-radio-btn.vue';
```

Comment:
```suggestion
export { default } from './ec-radio-btn.vue';
export * as types from './types';
```
Please also rename this file to .ts.